pkg/testutil/testutil.go

```diff
@@ -232,6 +232,14 @@ func (b *Base) InspectContainer(name string) dockercompat.Container {
 	return dc[0]
 }
 
+// Check if HyperV is set in the container
+// The flag is not yet part of the dockercompat spec
+func (b *Base) InspectContainerModeNative(name string) string {
```

Comment:
This should return the struct from https://github.com/containerd/nerdctl/tree/main/pkg/inspecttypes/native

Reply:
Removed the function, doing the inspect output test in the test function.